### DISCUSSION TITLE
v2/components/zh/config

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#405] [Announcements]
# v2/components/zh/config

components 类型: AvailableComponent[]; 默认值: []; 需要被注册的组件。 可接受的组件名称为: 

https://vuepress-theme-hope.github.io/v2/components/zh/config.html

**Comment by rocky-peng** [comment]
pdf没法预览

**Reply by Mister-Hope** [reply]
如果你觉得有问题，提供mininal reproduction ，开一个bug单。
